R/GQDesign.R

```diff
@@ -0,0 +1,81 @@
+setClass("GQDesign", representation(
+        n1        = "numeric",
+        c1f       = "numeric",
+        c1e       = "numeric",
+        n2_pivots = "numeric",
+        c2_pivots = "numeric",
+        knots     = "numeric",
+        weights   = "numeric"
+    ),
+    contains = "Design")
+
+GQDesign <- function(n1, c1f, c1e, n2_pivots, c2_pivots, knots, weights) {
```

Comment:
... this means that the constructor only takes 'order' as integer input, constructs the GQ rule via

    rule <- gaussquad::legendre.quadrature.rules(order)[[10]]